test/e2e/transformer/test_transformer_odh.py

```
@@ -0,0 +1,535 @@
+
+def _log_pod_status(kserve_client, service_name):
+    """Log pod status for debugging test failures."""
+    try:
+        pods = kserve_client.core_api.list_namespaced_pod(
+            KSERVE_TEST_NAMESPACE,
+            label_selector=f"serving.kserve.io/inferenceservice={service_name}",
+        )
+        for pod in pods.items:
+            logger.info("Pod: %s  Phase: %s", pod.metadata.name, pod.status.phase)
+    except Exception:
+        pass
```

Ruff (0.15.17)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)